src/meson.build

```diff
@@ -31,7 +31,14 @@ if get_option('enable_d3d9')
   subdir('d3d9')
 endif
 
+if get_option('enable_d3d8')
+  if not get_option('enable_d3d9')
+    error('D3D9 is required for D3D8.')
+  endif
+  subdir('d3d8')
+endif
+
 # Nothing selected
-if not get_option('enable_d3d9') and not get_option('enable_dxgi')
+if not get_option('enable_d3d8') and not get_option('enable_d3d9') and not get_option('enable_dxgi')
```

Comment:
this seems redundant? not d3d9 -> not d3d8 since we'd error out otherwise just above.